### PR TITLE
chore: withdrawEarlyUnbondedTransaction output index

### PR DESCRIPTION
**Pull request by gbarkhatov**
package.lock needs to be updated (`npm i`) once the library has needed version

### CODE REVIEW

**Comment by vitsalis** [comment]
Maybe we should merge this PR only when a non-canary version of btc-staking-ts has been released?

**Comment by gbarkhatov** [comment]
@vitsalis @jrwbabylonchain 
updated to `btc-staking-ts` `0.2.9`, `package.lock` also updated `simple-staking` `package.json` to `0.2.8`

**Comment by gbarkhatov** [comment]
@jrwbabylonchain yes

**Comment by gbarkhatov** [comment]
@jrwbabylonchain the `btc-staking-ts` change is still not on `main`

File: package.json [hunk 1]
```diff
@@ -30,7 +30,7 @@
     "@tanstack/react-query-next-experimental": "^5.28.14",
     "axios": "^1.6.8",
     "bitcoinjs-lib": "^6.1.5",
-    "btc-staking-ts": "^0.2.8",
+    "btc-staking-ts": "^0.2.9",
```
[the remark this answers is not in the document]
Comment (gbarkhatov) [reply package.json:33]: @jrwbabylonchain main `0.2.10` has optional index, as we had before. Dev version has no such parameter for manually unbonded tx, the whole purpose of this PR in `simple-staking` is to get rid of it's usage

[the remark this answers is not in the document]
Comment (gbarkhatov) [reply package.json:33]: Okay so the decision is to use `0.2.10` and not pass an arg even if it's option in `0.2.10`, correct?

**Pull request merged**